setup.py

```diff
@@ -30,19 +30,19 @@
         raise
 
 # configure the env
-libraries = ['SDL', 'SDL_mixer']
+use_sdl2 = os.environ.get('USE_SDL2')
+libraries = ['SDL2', 'SDL2_mixer'] if use_sdl2 else ['SDL', 'SDL_mixer']
 library_dirs = []
-include_dirs = ['/usr/include/SDL']
+include_dirs = ['/usr/include/SDL2', '/home/kivy/.buildozer/android/platform/android-ndk-r9c/sources/android/support/include'] if use_sdl2 else ['/usr/include/SDL2']
```

Comment:
Same as before, use the NDK folder directly in P4A recipe, not here :) If you need, just write the raw path to the recipe and check with @inclement how to get the right path, however I think it's somewhere in the `self.ctx` part as I said before.